packages/PlanLimitsUI/src/pages/Limits/RiverTilesSource.tsx

```diff
@@ -0,0 +1,66 @@
+import { Layer, Source } from 'react-map-gl';
+
+function determineTileServerHost(hostname: string) {
+  switch (hostname) {
+    case 'plan-limits.gw-eop-dev.tech':
+      return 'https://tiles.gw-eop-dev.tech';
+    case 'plan-limits.gw-eop-stage.tech':
+      return 'https://tiles.gw-eop-stage.tech';
+    case 'app.eop.gw.govt.nz':
+      return 'https://tiles.eop.gw.govt.nz';
+    default:
+      return 'http://127.0.0.1:7800';
```

Comment:
nit: should this be local host ...or should we make the `api/index.ts` also be 127.0.0.1